apps/condo/index.js

```diff
@@ -143,6 +144,64 @@ const apps = () => {
 
 /** @type {(app: import('express').Application) => void} */
 const extendExpressApp = (app) => {
+    const runtimeStats = {
+        activeRequestsIds: new Set(),
+        activeRequestsCountByType: {
+            graphql: 0,
+            oidc: 0,
+            other: 0,
+        },
+    }
+
+    app.use(function runtimeStatsMiddleware (req, res, next) {
+        const url = req.url
+        const method = (req.method || 'get').toLowerCase()
+
+        const isPost = method === 'post'
+
+        const isAdminApiUrl = url === '/admin/api'
```

Comment:
This one will not work in case like `/admin/api?deleted=true` as far as I remember, since req.url contains query params
<img width="805" alt="image" src="https://github.com/user-attachments/assets/ab7acacc-672b-4fbd-aeda-73e53ffa12f0" />


Consider using (new URL + pathname) option as I mentioned before